### ISSUE TITLE
display_link parameter in get_storm_data() does not function correctly

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#17]
Messages are still being displayed.

**Comment by timtrice** [comment]
Poor issue on my part. No idea what this is referencing. Best guess is using the example, 

```r
get_storm_data("public", 
    link = "http://www.nhc.noaa.gov/archive/1998/1998CHARLEYadv.html")
```

Messages are displayed showing the advisory being "worked":

```
Working http://www.nhc.noaa.gov/archive/1998/archive/pub/PAL0398.001
Working http://www.nhc.noaa.gov/archive/1998/archive/pub/PAAL0398.001
Working http://www.nhc.noaa.gov/archive/1998/archive/pub/PAL0398.002
Working http://www.nhc.noaa.gov/archive/1998/archive/pub/PAAL0398.002
Working http://www.nhc.noaa.gov/archive/1998/archive/pub/PAL0398.003
Working http://www.nhc.noaa.gov/archive/1998/archive/pub/PAL0398.004
Working http://www.nhc.noaa.gov/archive/1998/archive/pub/PAAL0398.004
Working http://www.nhc.noaa.gov/archive/1998/archive/pub/PAL0398.005
[1] TRUE
```

Will add parameter to the call to hide these messages and set to TRUE by default.

**Comment by timtrice** [comment]
Resolved in commit 1566c18
